src/SchemaGenerator.ts

```diff
@@ -167,6 +167,25 @@ export class SchemaGenerator {
             case ts.SyntaxKind.ArrowFunction:
                 allTypes.set(`NamedParameters<typeof ${this.getFullName(node, typeChecker)}>`, node);
                 return;
+            case ts.SyntaxKind.ExportSpecifier: {
+                const exportSpecifierNode = node as ts.ExportSpecifier;
+                const symbol = typeChecker.getExportSpecifierLocalTargetSymbol(exportSpecifierNode);
+                if (symbol?.declarations?.length === 1) {
```

Comment:
Not sure if `declarations` can ever have more than 1 element, and in what case. The `=== 1` check is going to prevent unexpected behavior, but there may be edge cases we _should_ handle – let me know